spring-cloud-gcp-logging/README.adoc

```diff
@@ -1,74 +1 @@
-}
-----
-
-https://cloud.google.com/logging/[Stackdriver Logging] is the managed logging service provided by Google Cloud
-Platform.
-
-This module provides support for associating a web request trace ID with the corresponding log entries.
-This allows grouping of log messages by request.
-
-NOTE: Due to the way logging is set up, the GCP project ID and credentials defined in
-`application.properties` are ignored.
-Instead, you should set the `GOOGLE_CLOUD_PROJECT` and `GOOGLE_APPLICATION_CREDENTIALS` environment
-variables to the project ID and credentials private key location, respectively.
-
-`TraceIdLoggingWebMvcInterceptor` extracts the request trace ID from an HTTP request using a
-`TraceIdExtractor` and stores it in a thread-local of the https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-logging/src/main/java/com/google/cloud/logging/TraceLoggingEnhancer.java[`TraceLoggingEnhancer`]
-which can then be used in a logging appender to add the trace ID metadata to log messages.
-
-There are implementations provided for `TraceIdExtractor`:
-
-[options="header",]
-|=======================================================================
-| Extractor | Description
-| `XCloudTraceIdExtractor` | Checks the `X-Cloud-Trace-Context` HTTP header that is automatically added to HTTP requests sent to applications hosted on GCP services such as App Engine and GKE
-| `ZipkinTraceIdExtractor` | Checks the `X-B3-TraceId` header
-| `CompositeTraceIdExtractor` | Instantiated with a list of other `TraceIdExtractor`, and provides the first trace ID found using these extractors in the given order
-|=======================================================================
-
-`LoggingWebMvcConfigurer` configuration class is also provided to help register the `TraceIdLoggingWebMvcInterceptor`
-in Spring MVC applications.
-
-Currently, https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-logging#add-a-stackdriver-logging-handler-to-a-logger[Java Util Logging (JUL)]
-and https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-contrib/google-cloud-logging-logback[Logback] are supported.
-
-== Logback Support
-
-For Logback, a `org/springframework/cloud/gcp/logging/logback-appender.xml` file is made available for import to make it
-easier to configure the Logback appender.
-
-Your configuration may then look something like this:
-[source, xml]
-----
-<configuration>
-  <include resource="org/springframework/cloud/gcp/logging/logback-appender.xml" />
-
-  <root level="INFO">
-    <appender-ref ref="STACKDRIVER" />
-  </root>
-</configuration>
-----
-
-`STACKDRIVER_LOG_NAME` and `STACKDRIVER_LOG_FLUSH_LEVEL` environment variables can be used to customize the
-`STACKDRIVER` appender.
-
-Also see the link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging[spring-cloud-gcp-starter-logging] module.
+include::../docs/src/main/asciidoc/logging.adoc[]
```

Comment:
We already have an issue tracking the creation of the reference documentation for Logging. See #288.
I would rather handle that separately, and limit this PR to extending this README instead.

Reply:
I would prefer to stay away of any work related to READMEs at all...

Reply:
Ultimately, yes, but for now, I think it's OK.

Reply:
So what to do now? @artembilan told me to add the doc to the reference manual and @meltsufin  tell me to move it back to the README. As the readme will link to the chapter of the reference manual, I think we are fine.

Reply:
Well, since we have that issue for Docs.
Let's just don't anything on the matter in this PR!

I think you are free to pick that up and PR the fix for Docs separately.